modules/caddyhttp/fileserver/matcher.go

```diff
@@ -480,7 +480,7 @@ func (m MatchFile) selectFile(r *http.Request) (matched bool) {
 		return true
 	}
 
-	return
+	return false
```

Comment:
Actually you might want to also  remove the named return above in `func (m MatchFile) selectFile(r *http.Request) (matched bool)`  since it seems completely unused except for the implicit `false` values which you just replaced.

It would probably help with readability, but would require to also add to the description to say "returns whether it matched or not".